src/quickfix.c

```diff
@@ -6612,7 +6612,7 @@ ex_vimgrep(exarg_T *eap)
 	goto theend;
     }
 
-    // Jump to first match if the current window is not 'winfixbuf'
+    // Jump to first match.
```

Comment:
This comment also seems wrong as 'winfixbuf' is checked at the top of this function.